### PR TITLE
Roadmap 2020

### PR DESCRIPTION
**Pull request by jkutner**
Fleshed out the meat of the roadmap. added categories and prose

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -10,6 +17,7 @@ The Cloud Native Buildpacks project is rallying around the theme of _THEME_GOES_
 
 ### Reproducibility
 - Outcomes: Builds are easier to understand
```
Comment (Zanadar) [inline ROADMAP.md]: > Builds verifiable and build output can be replicated.

Comment (sclevine) [inline ROADMAP.md]: Reproducibility is really a security feature. How about "Builds are verifiable. Build output can be replicated."

File: ROADMAP.md [hunk 2]
```diff
@@ -9,7 +16,8 @@ The Cloud Native Buildpacks project is rallying around the theme of _THEME_GOES_
 - Issue(s: https://git.io/JvJGt
```
Comment (nebhale) [inline ROADMAP.md]:
```suggestion
- Issue(s): https://git.io/JvJGt
```

File: ROADMAP.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # Roadmap
-The Cloud Native Buildpacks project is rallying around the theme of _THEME_GOES_HERE_ for  our upcoming work in 2020. Below are outcomes and features we intend to explore in the coming months. This roadmap will continue to be updated as our priorities evolve.
+
+The Cloud Native Buildpacks project is rallying around the theme *Coming of Age** in 2020. Our roadmap will emphasize features and fixes that bring Cloud Native Buildpacks toward a 1.0 release, signaling that the project is mature enough for anyone to use it in production.
```
Comment (ekcasey) [inline ROADMAP.md]:
```suggestion
The Cloud Native Buildpacks project is rallying around the theme *Coming of Age* in 2020. Our roadmap will emphasize features and fixes that bring Cloud Native Buildpacks toward a 1.0 release, signaling that the project is mature enough for anyone to use it in production.
```

File: ROADMAP.md (hunk 2 above)
Comment (ekcasey) [inline ROADMAP.md]:
```suggestion
- Issue(s): https://git.io/JvJGt
```

**Pull request merged**